src/components/demo/shapesdemos.tsx

```diff
@@ -12,7 +12,8 @@ import {
   Hexagon,
   Polygon,
   Star,
-  Cross
+  Cross,
+  SquaredDonut
```

Comment:
```suggestion
  SquareDonut
```
Make the required changes in other places as well.